virttest/nfs.py

```diff
@@ -332,6 +332,32 @@ def setup_remote(self):
                 raise exceptions.TestFail("Failed to run %s: %s" % (mkdir_cmd, o))
             self.mkdir_mount_remote = True
 
+        # Check firewall in host permits nfs service to mount nfs directory
+        # from remote machine
+        try:
+            firewall_cmd = "firewall-cmd --list-all | grep services:"
```

Comment:
@will-Do okay, then I will use iptables if it excepts with firewall-cmd to handle it for RHEL6 based systems